bigbluebutton-html5/imports/ui/components/logout-confirmation/component.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router';
 import { defineMessages, injectIntl } from 'react-intl';
+import { makeCall } from '/imports/ui/services/api';
+import Button from '/imports/ui/components/button/component';
 import Modal from '/imports/ui/components/modal/fullscreen/component';
+import Auth from '/imports/ui/services/auth';
+import userListService from '../user-list/service';
```

Comment:
@oswaldoacauan is it right?
I mean, as far as I remember we agreed that only "service" can communicate with the Api, but if there is no need in reactive data from the container - why having a container at all in this case?